scripts/avalanchego-installer.sh

```diff
@@ -363,6 +413,25 @@ create_config_file
 create_service_file
 chmod 644 avalanchego.service
 sudo cp -f avalanchego.service /etc/systemd/system/avalanchego.service
+
```

Comment:
Why this duplicate block? Forgotten scratch space?

Reply:
this was a copy and paste error. Duplicate block can be deleted.